problem5.py

```diff
@@ -0,0 +1,76 @@
+#problem5
+#the smallest number that can be divided by each of the numbers from 1 to x without any remainder    
+import math
+
+def prime_if(max_number):
+# prime factors
+    i = 0
+    for divider in range(2, max_number):
+        if max_number % divider == 0:
```

Comment:
Why you are not importing this function from is_prime module?

Reply:
It was before I used this function more than once. Thaks